css/ampshared.css

```diff
@@ -539,17 +539,24 @@ amp-accordion {
   display: block !important;
 }
 
-/* heading */
- .i-amphtml-accordion-header {
-  margin: 0;
-  cursor: pointer;
-  background-color: #efefef;
-  padding-right: 20px;
-  border: solid 1px #dfdfdf;
-}
-/* heading/content */
-.i-amphtml-accordion-content {
-  margin: 0;
+/* 
+This media query is used instead of @supports due to low compatibility.
+At the time of this change (4/6/2021) no version of Webkit support @supports queries.
+
+These provide low-specificity UI styles to preventing CLS for 
+the header and content segments of accordions.
+*/
+@media (min-width: 1px) {
+  :where(amp-accordion > section) > :first-child {
+    margin: 0;
+    cursor: pointer;
+    background-color: #efefef;
```

Comment:
nit: might be fine to leave out the cursor and background color for the async styles, shave a tiny chunk of bytes off the shared boilerplate

Reply:
Interesting, will take a look at the delta.

Reply:
This also sets a default border size, so we'd need to retain two selectors to not apply the default border to the `last-child`.

As a result the reduction would be omitting `cursor:pointer;background-color:#efefef;` from `ampshared.css` which seems less valuable.

Let's do this in a different PR if we think it's helpful.

Reply:
A benefit of keeping `background-color` is that it reduces LCP. 

Left is with background-color commented out (and patch inlined as `style[amp-custom]`), and right is with the entire  `amp-accordion.css` inlined:

https://user-images.githubusercontent.com/134745/113763708-d7a0c880-96ce-11eb-9118-128783b8ce83.mov

From https://amp-accordion-layout-shifting.glitch.me/

That being said, `cursor:pointer` can definitely go since before the runtime loads, a pointer cursor would be misleading and rage clicking could ensure.